api/lib/domain/validators/session-validator.js

```diff
@@ -59,9 +60,16 @@ module.exports = {
   },
 
   validateFilters(sessionFilters) {
-    const { error } = sessionFiltersValidationSchema.validate(sessionFilters, validationConfiguration);
+    const filters = _.clone(sessionFilters);
+    _.each(filters, (value, key) => {
+      if (typeof filters[key] === 'string') {
+        filters[key] = filters[key].trim() || undefined;
+      }
+    });
```

Comment:
```suggestion
    _.each(filters, (value, key) => {
      if (typeof value === 'string') {
        filters[key] = value.trim() || undefined;
      }
    });
```

Reply:
Il y a sans doute mieux dans lodash, un mapValues ou autre, je regarde.

Reply:
Peut-être ça, plus besoin du `_.clone()` :
```suggestion
    const filters = _.mapValues(sessionFilters, (value) => {
      if (typeof value === 'string') {
        return value.trim() || undefined;
      }
      return value;
    });
```

Reply:
Merci !